composer.json

```diff
@@ -15,15 +15,15 @@
     "require": {
         "php": ">=5.5",
         "zendframework/zend-eventmanager": "dev-develop as 2.7.0",
-        "zendframework/zend-stdlib": "~2.7"
+        "zendframework/zend-stdlib": "dev-develop as 2.8.0"
     },
     "require-dev": {
         "zendframework/zend-config": "dev-develop as 2.6.0",
         "zendframework/zend-console": "~2.5",
-        "zendframework/zend-di": "~2.5",
+        "zendframework/zend-di": "dev-develop as 2.6.0",
         "zendframework/zend-loader": "~2.5",
-        "zendframework/zend-mvc": "~2.5",
-        "zendframework/zend-servicemanager": "dev-develop as 2.6.0",
+        "zendframework/zend-mvc": "dev-develop as 2.7.0",
+        "zendframework/zend-servicemanager": "dev-develop as 2.7.0",
```

Comment:
No, it has to be `^2.7 || ^3.0`, so that it can be pulled into either codebase. That said, still working on a comprehensive set of changes we can use.